### PR TITLE
Created template for security related report

### LINKED ISSUE #168: Issue template for security-related reports
Provide an issue template for security-related reports such as vulnerabilities.

### PR DESCRIPTION
**Pull request by Hekku2**
Security issue template

Fixes #168 

## Proposed Changes

  - Added new template for security issues

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Security_issue.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+name: Security issue
+about: Report security issue affecting this project
+
```
Comment (tomkerkhove) [inline .github/ISSUE_TEMPLATE/Security_issue.md]: Please remove this empty line

File: .github/ISSUE_TEMPLATE/Security_issue.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Not always applicable
+ 1. Step 1
+ 2. Step 2
+ 3. Step ...
+
+## Migitation
+
+Migitation options, if there are any
+
+## Additional resources
```
Comment (tomkerkhove) [inline .github/ISSUE_TEMPLATE/Security_issue.md]: Let's rename this to "Vulnerability Information" and move it before "Steps to reproduce"

File: .github/ISSUE_TEMPLATE/Security_issue.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Link to National Vulnurability Database or similat (CVE-2018-14041)[https://nvd.nist.gov/vuln/detail/CVE-2018-14041]
+
+## Specification
```
Comment (tomkerkhove) [inline .github/ISSUE_TEMPLATE/Security_issue.md]: Let's rename this to "Application Specifications"

File: .github/ISSUE_TEMPLATE/Security_issue.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+- Version(s):
```
Comment (tomkerkhove) [inline .github/ISSUE_TEMPLATE/Security_issue.md]: Let's make this "Docker Image Version"

**Comment by tomkerkhove** [comment]
Looks good @Hekku2, have a few small suggestions and we can merge this in.
Thanks for jumping in!

**Comment by Hekku2** [comment]
Hopefully this commit addresses those issues. I tried that [ci skip]-thing this time.

**Comment by tomkerkhove** [comment]
Perfect, thank you!

**Pull request merged**